internal/pkg/cli/env_upgrade.go

```diff
@@ -55,6 +69,10 @@ func newEnvUpgradeOpts(vars envUpgradeVars) (*envUpgradeOpts, error) {
 
 		store: store,
 		sel:   selector.NewSelect(prompt.New(), store),
+		envTemplate: stack.NewEnvStackConfig(&deploy.CreateEnvironmentInput{
```

Comment:
Should this be named as
```suggestion
		legacyEnvTemplater: stack.NewEnvStackConfig(&deploy.CreateEnvironmentInput{
```
What do you think?

Reply:
👍 great suggestion!